agenthub/codeact_agent/codeact_agent.py

```diff
@@ -228,7 +228,7 @@ def step(self, state: State) -> Action:
         except OperationCancelled as e:
             raise e
         except Exception as e:
-            logger.error(f'{e}')
+            logger.error(f'LLM exception: {str(e)}')
```

Comment:
Just a heads-up, I merged the revert of this Exception. I really think it will prove simpler to work with, in practice.